config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml

```diff
@@ -20,6 +20,7 @@ spec:
   image:
     name: flink:1.8.2
   jobManager:
+    accessScope: Cluster
```

Comment:
Why update this file? If not necessary, I'd suggest revert it.

Reply:
This is a required field, without this, the admission denied it. Though this is not related to this, i fixed it by hand

Reply:
···
              required:
              - accessScope
···

Reply:
I checked, in the CRD doc, it says optional, but in the go struct it is required. So it seems that we intended it to be optional, but somehow the implementation was not aligned. But it is weird that in my test it doesn't fail without adding it. Any idea?